content/news/buzz/2024q2-roadmap/contents.lr

```diff
@@ -0,0 +1,45 @@
+title: 2024Q2 Roadmap
+---
+author: Russell Keith-Magee
+---
+body:
+
+Q1 2024 has seen some major progress - but as always, there's still plenty of work to do. As always, this roadmap should be read as a guide to what we aim to focus on over the coming quarter, rather than a hard commitment of features that will be made available on a specific deadline.
+
+Q1 progress
+-------------
+
+The first quarter of 2024 has seen significant progress towards getting iOS and Android to Tier 3 supported status in CPython. `PEP 738 <https://peps.python.org/pep-0738/>`__ as been formally accepted by the CPython steering committee; and the vast majority of the changes needed to support  `iOS <https://github.com/python/cpython/issues/114099>`__ and `Android <https://github.com/python/cpython/issues/116622>`__ have either been merged, or received substantial review from the CPython core team. 
+
+We've also made progress on a number of long standing feature requests in Toga. We've added an OptionContainer for Android, so tab-based UIs can be used on all platforms. We've added a MapView widget that works on all desktop and mobile platforms. We've also added the start of hardware APIs, including Camera and Geolocation APIs.
+
+Q2 priorities
+--------------
+
+Our Q2 goals are:
+
+* **Get iOS and Android to Tier 3 supported status**. We expect to finish upstreaming the patches we have for CPython early in the quarter. However. Tier 3 status also requires CI infrastructure to verify that the platforms are stable. In Q2, we will commission buildbots so that CPython's CI is able to verify the ongoing stability of iOS and Android as a platform.
```

Comment:
```suggestion
* **Get iOS and Android to Tier 3 supported status**. We expect to finish upstreaming the patches we have for CPython early in the quarter. However, Tier 3 status also requires CI infrastructure to verify that the platforms are stable. In Q2, we will commission buildbots so that CPython's CI is able to verify the ongoing stability of iOS and Android as a platform.
```